src/lms-ds-loader/.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 110
```

Comment:
We are using 88 for other projects... We should define a general line length.

Reply:
110 is the accepted standard in the Ed-Fi coding conventions.